package.json

```
@@ -27,7 +27,7 @@
   },
   "homepage": "https://github.com/alastria/alastria-identity-example#readme",
   "dependencies": {
-    "alastria-identity-lib": "github:alastria/alastria-identity-lib#master",
+    "alastria-identity-lib": "github:alastria/alastria-identity-lib#feature/issue-118-interfaces",
```

```suggestion
    "alastria-identity-lib": "github:alastria/alastria-identity-lib#develop",
```
Change this line before merge